src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/DiagnosticPortConfigurations/Connect.txt

```diff
@@ -0,0 +1,4 @@
+"DiagnosticPort": {
```

Comment:
Note that these are txt files. This was done because valid JSON would require the opening and closing braces, which would then need to be removed to do the `contains` check. I left it this way because they're small snippets, but this can be changed if we'd rather not use this approach.